x-pack/legacy/plugins/siem/public/pages/case/components/__mock__/router.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Router } from 'react-router-dom';
+/* eslint-disable @kbn/eslint/module_migration */
```

Comment:
nit: Is it possible to disable only for this line?

Reply:
it already is :)

Reply:
ahhh i see what you mean, to use the eslint-disable-next-line, will do